src/Middleware/HeaderPropagation/src/HeaderPropagationClientOptions.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.HeaderPropagation
+{
+    /// <summary>
+    /// Provides configuration for the <see cref="HeaderPropagationMiddleware"/>.
+    /// </summary>
+    public class HeaderPropagationClientOptions
```

Comment:
`HeaderPropagationMessageHandlerOptions`?

Reply:
What about `HeaderPropagationOptions`? is it ok to leave it like this as in the happy path you only deal with it? or should it be renamed to `HeaderPropagationMiddlewareOptions`?